src/module/actor/loot/sheet.ts

```diff
@@ -84,6 +86,65 @@ export class LootSheetPF2e<TActor extends LootPF2e> extends ActorSheetPF2e<TActo
         data.hidden = isMerchant && item.isOfType("treasure") && item.isCoinage && !item.container;
         return data;
     }
+
+    private async transferToPartyStash(): Promise<void> {
+        const activeParty = game.actors.party;
+
+        if (!activeParty) {
+            ui.notifications.warn("No active party found.");
+            return;
+        }
+
+        const newItems: PhysicalItemPF2e[] = [];
+        const itemUpdates = new Map<string, number>();
+        const itemsToDelete: string[] = [];
+
+        for (const item of this.actor.inventory) {
+            const stackableItem = activeParty.inventory.findStackableItem(item);
+            if (stackableItem) {
+                const currentQuantity = itemUpdates.get(stackableItem.id) ?? stackableItem.quantity;
+                itemUpdates.set(stackableItem.id, currentQuantity + item.quantity);
+                continue;
+            }
+            newItems.push(item);
```

Comment:
For now, and only right now, could this be a helper (actor/helpers.ts) is fine to move multiple items, and refactor the thing you copied this from to use that method? This is all going to need a refactor, and it'll be easier if we have it all in a single place. Moving items in bulk is going to be needed if we ever create a trade window anyways.

Reply:
@CarlosFdez thanks for raising this and pointing out where the helpers live.  I really didn't like copy/pasting to begin with and wanted to make a utility, but had no idea where it should go!

Reply:
To stress, the utility is completely temporariy. I plan to kill it all soon, but so long as the crime doesn't get *worse* I think its ok.